src/pages/jinminYang/Login/Login.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import "./Login.scss";
+import { Link } from "react-router-dom";
+import { useState } from "react";
+
+const Login = () => {
+  const [id, setId] = useState(null);
+  const [pw, setPw] = useState(null);
```

Comment:
```suggestion
  const [id, setId] = useState("");
  const [pw, setPw] = useState("");
```
해당 State에는 문자열이 들어올 것이기 때문에 초기값을 빈 문자열로 두는 것이 더 좋은 방법입니다!